### ISSUE TITLE
Method API documentation should not be repeated in docs

### ISSUE THREAD

**Issue by Teyras** [Cognexa/cxflow#107]
For example https://github.com/Cognexa/cxflow/blob/dev/docs/advanced/config.rst#main-loop - parameters should be documented in the source and included in the "human" documentation, either using an ordinary link or some sort of autodoc directive.

**Comment by blazekadam** [comment]
You can use autodoc directives to generate the docs in the tutorials. E.g.:
```
.. autoclass:: Noodle
   :members:
```

these directives are actually generated by the `autoapi` module:
```
.. autoclass:: AbstractDataset
   :show-inheritance:
   :members:
   :private-members:
   :special-members:
   :exclude-members: __weakref__

   .. inheritance-diagram:: AbstractDataset
      :parts: 1

   |
```

Further reading: http://www.sphinx-doc.org/en/stable/ext/autodoc.html

**Comment by blazekadam** [comment]
This is resolved in 0.8.2
